content/source/docs/enterprise/getting-started/policies.html.md

```diff
@@ -0,0 +1,76 @@
+
+A policy consists of:
+
+- The Sentinel policy code
+- An enforcement mode that changes how a policy affects the run lifecycle
+
+Currently, policies in an organization apply to **all** workspaces. You'll want to consider that when writing your policies so they won't fail unrelated workspace runs or configurations. In the future, TFE will provide a way to manage policies so you can apply them per-workspace.
+
+## Creating a Policy
+
+First, make sure you're viewing the organization settings. If you're still on a run page (or any other page), click the "Settings" button in the top navigation bar.
+
+![navigate](./images/policy-navigate.png)
+
+The policy list shows all of the policies you have access to; if you haven't created any, it's empty.
+
+To create your first policy, click the "Create new policy" button in the upper right.
+
+![new button](./images/policy-new-button.png)
+
+On the "Create a new Policy" page, you need to enter at least two items: a policy name, and the policy code. When you've finished, click the "Create Policy" button.
+
+![fields on new policy page](./images/policy-new-fields.png)
+
+### Policy Name
+
+A policy name should tell your colleagues what the policy is for. Most policies are _a particular environment_ of _a particular Terraform configuration,_ so the name should include both the name of the configuration and the name of the environment.
+
+For this example, we'll just create a sample passthrough policy that will allow all runs to "PASS" our policy check. 
+
+In this example, we're using a configuration named "minimum" and we're deploying it in a production environment, so we named it `passthrough`.
+
+### Enforcement Mode
+
+Enforcement Mode alters how a policy result affects your run; "hard-mandatory" will always stop a run if a policy fails, "soft-mandatory" will pause a run and allow a failure to be overridden, and "advisory" will log failures but always allow a run to continue. Use "soft-mandatory" for now so we can see
+
+### Policy Code
+
+You can paste the following code into the code input box. It always resolves to `true` and will allow all runs to pass. This is a good way to see policy checks applied to your run without having a policy "FAIL".
+
+```python
+main = rule { true }
+```
+
+Later, you can switch `true` to `false` to see how a "FAIL" during a policy check can affect your run. You can find many more examples in our [Example policies doc](../sentinel/examples.html).
+
+## What Happens in a New Policy
+
+When you create a new policy, it will be applied to all future runs. Runs that are currently queued or in progress will not be affected.
+
+## See a Policy Check in a Run
+
+Once your policy is created, you can view its effect on the run page. Start a new run to watch it play out. Again, you can start a run with the "Queue Plan" at the upper right of the workspace page, or using the "Save & Plan" button when editing the workspace's variables.
```

Comment:
> with the "Queue Plan" at the upper right of the workspace page

Should probably be changed to "with the "Queue Plan" _button_ at the upper right of the workspace page"